build/webpack/webpack.extension.web.config.js

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+'use strict';
+
+const path = require('path');
+const tsconfig_paths_webpack_plugin = require('tsconfig-paths-webpack-plugin');
+const webpack = require('webpack');
+const constants = require('../constants');
+const CleanTerminalPlugin = require('clean-terminal-webpack-plugin');
```

Comment:
This is the webpack config for the web output of the extension.

It generates:
- extension.web.bundle.js
- test/vscode.test/index.bundle.js
- test/smoke.test/index.bundle.js

The test bundles shouldn't be included in the extension, but are necessary for running tests.

Reply:
Out of curiousity, why do we need a bundle for the web tests? Just compiling them using typescript should suffice.

That would resolve the Symbol issue with IApplicationShell & the like.

Reply:
Same for smoke tests, 
i.e. creating a bundle is useful for publishing else i don't think we need to create a bundle.

Reply:
We need a bundle so the test server can load it. They get loaded into a web context too.

Reply:
Or is it because the vscode runtime cannot load a non-bundled (split bundle) JS file?

If thats the case then this makes sense, we could run into other issues as well.
As we'd get types & objects imported incorrectly.
E.g. we coudl be referencing a class in test/index.bundle.js, but that would actually be referencing a class in the test bundle and not the actually extension bundle, & that would cause a lot of issues with the tests.

Reply:
E.g. things like ts-mockito and a lot of those test concepts will not work